src/components/Signin/Signin.jsx

```diff
@@ -68,7 +69,13 @@ function Signin() {
           <div className='mb-4 text-xl text-[#647581]'>
             <p>Don’t have an account??</p>
           </div>
-          <Button className='orangeSignIn ' text='Sign-up' />
+          {/* <Button className='orangeSignIn ' text='Sign-up' /> */}
```

Comment:
This needs a discussion

Reply:
i think the discussion will be with abdulrhman i just worked in the routing not in the component it self 😅